### DISCUSSION TITLE
op_cafeteriachair doesnt have chair tag (bots dont sit in them)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14492] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

op_cafeteriachair doesnt have chair tag, so bots don't naturally try to sit in them

Conversely, opdeco_wheelchair2 has chair tag, which means bots love sitting in them

_This came to mind after wondering why outpost NPCs would use chairs, but rarely interact with other interactables like cabinets, fabricators, reactor... This would make outposts more alive_

https://github.com/FakeFishGames/Barotrauma/blob/0aee53976202c7ea5c0bb3f26669a97c8dc2400c/Barotrauma/BarotraumaShared/SharedSource/Characters/AI/Objectives/AIObjectiveIdle.cs#L379-L392

### Version

v1.5.9.1 (Summer Update Hotfix 2)

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/1042baeef7f9b75dc31820467698fbebd53520de
